plenum/server/node.py

```diff
@@ -1303,9 +1297,8 @@ def checkInstances(self) -> None:
         This method is called whenever a connection with a  new node is
         established.
         """
-        logger.debug("{} choosing to start election on the basis of count {} "
-                     "and nodes {}".format(self, self.connectedNodeCount,
-                                           self.nodestack.conns))
+        logger.info("{} choosing to start election on the basis of count {} and nodes {}".
```

Comment:
This method does nothing, so let's keep it debug.

Reply:
debug